### PR TITLE
2nd iCBT preparation - Fixes

### PR DESCRIPTION
**Pull request by Sunaries**
Minor fixes to mob names and additions. Will continue later with the Bramble story for double checking.

@imcgames 3 questions.

GitHub Logo 

Q1. Just like with the anvil item for equipment upgrade, it says '_PlayerName_ of Anvil. This Sorcerer skill has the same manner. Any idea how to change it to: _PlayerName_'s Anvil or _PlayerName_'s Salamion?

GitHub Logo

Q2. Has there been any progress made concerning this proposal? As suggested in this PR -------> https://github.com/Treeofsavior/EnglishTranslation/pull/1246

Q3. The story at https://treeofsavior.com/intro needs some minor tweaks. Here is the final version. I added "_Goddess_ _Vakarine's_ **evening** _stars_ (since she is the goddess of the evening star)

Austras Koks is only used in this story for 'Divine Tree', right? Because 'World Tree' and 'Divine Tree' are more mentioned in the game than Austras Koks.

> Blessed by Goddess Laima, the capital of the kingdom was a
> symbol of prosperity. Under the reign of a wise king and with the
> blessings of several Goddesses, happiness filled the lives of people.
> 
> Goddess Zemyna, the goddess of earth, blessed the land for
> good harvest every year. Goddess Gabija, the goddess of fire,
> taught people how to use fire for comfortable homes and warm
> food. Goddess Vakarine's evening stars helped travelers navigate their
> way and Goddess Ausrine guided the souls to rest in peace.
> The Goddesses were always with the people.
> 
> However, one day, prayers began to be left unanswered. At first,
> people did not take this seriously, believing the Goddesses would
> soon return. But that day may have been the warning.
> 
> Austras Koks suddenly emerged from the center of the capital
> and started to ravage the kingdom. Its enormous size destroyed
> the capital before anything could be done and many shed tears
> for the loss of their loved ones. People started fleeing, but the
> disaster was not something easily avoidable.
> 
> Refugees saw gruesome sights. Lush trees and splendid flowers 
> turned into monsters that swallowed people, and fields and forests
> turned into nightmares. The force that had ruined the capital
> was so great, that its damaging effect reached many other cities.
> 
> Medzio Diena.
> Four years have passed, but these words continue to haunt the
> people. Monsters continue to spread violence and fear while
> nobody knows when their once great city will be restored. 
> The Goddesses never returned.
> 
> Four years was certainly not enough to heal the wounds of people
> from that day. To this day, people try to find a way back to their
> past normal lives.
> 
> One day amidst despair, people from around the kingdom gath-
> ered in Klaipeda claiming that they had dreamed of the Goddess.
> Could this be a sign of the Goddesses' return? Or the beginning
> of a new disaster?
> 
> Nobody has an answer.

Thank you.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -9133,18 +9133,18 @@ ETC_20150317_009133	Ludas Meduza
...
 ETC_20150317_009136	Jonas
-ETC_20150317_009137	Tontus Smart Zen ROKAS24_SQ_03
-ETC_20150317_009138	Vakis of Adventurer
-ETC_20150317_009139	Bags of Adventurer
-ETC_20150317_009140	Soul of Adventurer Vakas
-ETC_20150317_009141	Hogma Combat Boss Smart Zen
+ETC_20150317_009137	Tontus Smart Gen ROKAS24_SQ_03
+ETC_20150317_009138	Adventurer Varkis
+ETC_20150317_009139	Adventurer's Bag
+ETC_20150317_009140	Adventurer Varkis Spirit
```
Comment (vtange) [inline ETC.tsv]: Varkis' Spirit

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by terrapin47** [comment]
:+1:

**Comment by vtange** [comment]
:+1:

**Comment by ttgmichael** [comment]
:+1: 10/10 xD

**Comment by ttgmichael** [comment]
Since @Sunaries have suggested reupdating the Main Synopsis on the Tree of Savior webpage (as it really could still be polished even further), here is my suggestions for what it could be:

Blessed by Goddess Laima, the kingdom's capital was a
symbol of prosperity. Under the reign of a wise king and with the
blessings of several Goddesses, happiness filled the lives of the people.

Goddess Zemyna, the goddess of earth, blessed the land with
bountiful harvest each year. Goddess Gabija, the goddess of fire, 
taught the people how to use fire to warm their homes and prepare hearty
meals. Goddess Vakarine's evening stars navigated travelers through their
journeys. And finally, Goddess Ausrine guided souls to rest in peace.
The Goddesses had always been with the people.

But one day, prayers began to be left unanswered. At first, the
people did not take this seriously, believing that the Goddesses would
soon return. No one suspected that the unanswered prayers to be foreboding.

The Divine Tree suddenly erupted from the center of the capital
and began ravaging the kingdom. Its colossal size devastated
the capital before anything could be done. The people attempted to flee, but the
menacing chaos engulfed them. Countless victims wept
for the loss of their loved ones. 

The refugees saw gruesome sights. Lush trees and radiant flowers 
turned into monsters that swallowed people, the fields and forests
turned into nightmares. The force which ruined the capital
was so great that its catastrophic effects struck distant cities.

Medzio Diena.
Four years have passed, but these words continue to haunt the
people. Monsters continue to spread violence and fear and
no one knows when their once great city will be restored. 
The Goddesses never returned.

Time has not passed long enough to heal the wounds of the victims who suffered
that day. To this day, there are still people trying to return back to their
former way of life.

Amidst the continuing despair, rumors from within the kingdom have arisen about people claiming that they had dreamt of the Goddess. Could this be a sign of the Goddesses' return? Or the beginning of a new disaster?

No one has an answer.

We hope to hear your opinions and answers to the questions @Sunaries has presented as well. ^^

**Comment by imcgames** [comment]
1. I agree with you. It looks more good display. but is it possible to change through only scripts? 
2. We're trying to apply this proposal. It will be updated with patch during iCBT, when completed.
3. Thanks. Regardless of the 2nd iCBT, it will be updated on the Forum later. (Actually, need more times to change it by requesting to the designer)

**Comment by imcgames** [comment]
:+1:

**Comment by Sunaries** [comment]
@ttgmichael Good synopsis, that can be used on the story page.

@imcgames I think it is located in the scripts, but I believe it requires a recode just like with Q2. However, I cannot find the 'of <PlayerName>' code in the scripts. Can you?

**Comment by ttgmichael** [comment]
nvm about the recode, we found the "of" 

ETC_20150317_001979 for future reference

**Pull request merged**